Dockerfile

```diff
@@ -4,9 +4,9 @@ LABEL maintainer="Chef Software, Inc. <docker@chef.io>"
 ARG EXPEDITOR_CHANNEL
 ARG CHANNEL=stable
 ARG EXPEDITOR_VERSION
-ARG VERSION=4.3.13
+ARG VERSION=3.10.1
```

Comment:
Title says `3.12` and it looks like that is correct. Or am I missing something?

Reply:
The promote will update this to whatever the latest is

Reply:
Still why would you modify this line if the promote will update it?